services/QuillLMS/spec/lib/rule_feedback_history_spec.rb

```diff
@@ -296,6 +296,7 @@ def feedback_factory(&hash_block)
         datetime: f_h.updated_at,
         entry: f_h.entry,
         highlight: f_h.metadata.instance_of?(Hash) ? f_h.metadata['highlight'] : '',
+        low_confidence_predicted_rule: {},
```

Comment:
Note that if you replace this key with a naive pass-through of `api` then this will also need to be updated.